src/LaunchDarkly.Client/Clause.cs

```diff
@@ -38,7 +38,8 @@ internal bool MatchesUser(User user)
                 {
                     if (!(element is JValue))
                     {
-                        Logger.LogError("Invalid custom attribute value in user object: " + element);
+                        Logger.LogError("Invalid custom attribute value in user object: {0}",
```

Comment:
Replace string instead of string concatenation

Reply:
Seems like these type of logging usage auditing is automated now. https://github.com/aspnet/Logging/pull/702